deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -813,6 +814,22 @@ def modify_metadata_vuldet_feed(feed, timestamp):
     control_service('start', daemon='wazuh-db')
 
 
+def modify_nvd_metadata_vuldet(timestamp):
+    """Function to modify the timestamp value of the nvd_metadata table.
+
+    Args:
+        timestamp (int): The new timestamp value to set.
+    """
```

Comment:
`Raises` field is missing

Reply:
Done in: https://github.com/wazuh/wazuh-qa/pull/1428/commits/1d7710d1cf9ff09af486d8ffdeb43e0b4dce8a9b